docker-compose.yml

```diff
@@ -0,0 +1,47 @@
+---
+version: "3.4"
+
+services:
+  app:
+    image: hargata/lubelog:latest
+    build: .
+    restart: unless-stopped
+    # volumes used to keep data persistent
+    volumes:
+      - config:/App/config
+      - data:/App/data
+      - documents:/App/wwwroot/documents
+      - images:/App/wwwroot/images
+      - log:/App/log
+    # expose port and/or use serving via traefik
+    ports:
+      - 8080:8080
+    # traefik configurations, including networks can be commented out if not needed
+    networks:
+      - traefik-ingress
```

Comment:
I'm thinking this should have a separate `docker-compose` file in the repo since not everyone will use this behind traefik.  I did see your comment about commenting out/deleting this part but it seems cleaner to me to have separate `yaml` files for specific configurations.  Up to @hargata, though!

Reply:
a separate yaml file would be fantastic, thanks!

Reply:
done. see pr #8  :)